### DISCUSSION TITLE
Could we get my turbowarp template added to the libraries section

### DISCUSSION THREAD

**Discussion by gamerlogan819** [MeowerBots/.github#9] [General]
This https://github.com/gamerlogan819/Meowerbot-Turbowarp

**Comment by daydreamyyZX** [comment]
no, zzbot has been refused so your turbowarp template will too

**Comment by JoshAtticus** [comment]
No, for 2 reasons

1. It's not a library, it's a template. You can't really make libraries for scratch unless it's an extension.
2. It's not part of the meower-community organisation.
